File: backend/src/main/java/org/multimes/backend/core/web/repository/interfaces/IDialogRepository.java
package org.multimes.backend.core.web.repository.interfaces;

import org.multimes.backend.core.web.model.entities.Dialog;

import java.util.List;

public interface IDialogRepository {
    public int add(Dialog dialog);

    public Dialog getById(int id);

    public int checkExistsWithIdInMessenger(long id);

    public List<Dialog> getAll();
}

File: backend/src/main/kotlin/org/multimes/backend/core/web/repository/interfaces/IDialogRepository.kt
package org.multimes.backend.core.web.repository.interfaces

import org.multimes.backend.core.web.model.entities.Dialog

interface IDialogRepository {
    fun add(dialog: Dialog): Int
    fun getById(id: Int): Dialog
    fun checkExistsWithIdInMessenger(id: Long): Int
    fun getAll(): List<Dialog>
}
